### PR TITLE
[T4A2][T13-B2] Gabriel Gheorghian

### PR DESCRIPTION
**Pull request by GabrielGhe**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,20 +11,23 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/CommandResult.java:14]: - This commit is unrelated to the concerned tutorial exercise.
- Avoid accumulating unrelated changes into a single PR. 
- Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -222,7 +222,7 @@ private int parseArgsAsDisplayedIndex(String args) throws ParseException, Number
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareFind(String args) {
+    private static Command prepareFind(String args) {
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java:225]: you will have to update the parser test file as well as it uses `parseCommand` method

**Comment by okkhoy** [comment]
@GabrielGhe 
Some comments added. Please ack & close the PR after reading comments.

**Comment by GabrielGhe** [comment]
Ack. Understood.

**Pull request closed without merging**